packages/types/src/lib/interfaces.ts

```diff
@@ -1055,34 +1081,35 @@ export interface GetSignSessionKeySharesProp {
   body: SessionRequestBody;
 }
 export interface CommonGetSessionSigsProps {
+  // If you want to pass the `authNeededCallback`, having a PKP public key is necessary. This can be used if you do not have a wallet and wish to authenticate with your PKP.
   pkpPublicKey?: string;
 
-  // When this session signature will expire.  The user will have to reauthenticate after this time using whatever auth method you set up.  This means you will have to call this signSessionKey function again to get a new session signature.  This is a RFC3339 timestamp.  The default is 24 hours from now.
+  // When this session signature will expire. After this time is up, you will need to reauthenticate using the same authentication method, generating a new session signature. The default time until expiration is 24 hours. The formatting is an [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) timestamp.
   expiration?: any;
 
-  //   The chain to use for the session signature.  This is the chain that will be used to sign the session key.  If you're using EVM then this probably doesn't matter at all.
+  // The chain to use for the session signature and sign the session key. If you're using EVM, this parameter isn't very important.
   chain?: Chain;
 
   /**
    * An array of resource abilities that you want to request for this session. These will be signed with the session key.
-   *
-   * @example If you want to request the ability to decrypt an access control condition, then you would pass
-   * [{ resource: new LitAccessControlConditionResource('someResource), ability: LitAbility.AccessControlConditionDecryption }]
+   * If you want to request the ability to decrypt an access control condition, then you would pass something similar to the example:
```

Comment:
> ...to decrypt an access control condition

Confusing to me, @Ansonhkg @MaximusHaximus are we actually decrypting the ACC when we call decrypt?

Reply:
So you can add an `ability` which will allow the `session` to decrypt content associated with the specific hash of the ACC conditions. So the conditions are not being decrypted. but the ability is looked up in the `resouceAbilityRequests` to figure out if the session has the ability to decrypt the content.